ext/pg.c

```diff
@@ -451,13 +451,32 @@ Init_pg_ext()
 
 	/******     PG::Connection CLASS CONSTANTS: Error Verbosity     ******/
 
-	/* Terse error verbosity level (#set_error_verbosity) */
+	/* Terse error verbosity level (#set_error_verbosity).
+	 * In TERSE mode, returned messages include severity, primary text, and position only; this will normally fit on a single line. */
 	rb_define_const(rb_mPGconstants, "PQERRORS_TERSE", INT2FIX(PQERRORS_TERSE));
-	/* Default error verbosity level (#set_error_verbosity) */
+	/* Default error verbosity level (#set_error_verbosity).
+	 * The DEFAULT mode produces messages that include the above plus any detail, hint, or context fields (these might span multiple lines). */
 	rb_define_const(rb_mPGconstants, "PQERRORS_DEFAULT", INT2FIX(PQERRORS_DEFAULT));
-	/* Verbose error verbosity level (#set_error_verbosity) */
+	/* Verbose error verbosity level (#set_error_verbosity).
+	 * The VERBOSE mode includes all available fields. */
 	rb_define_const(rb_mPGconstants, "PQERRORS_VERBOSE", INT2FIX(PQERRORS_VERBOSE));
 
+/* PQERRORS_SQLSTATE was introduced in PG-12 together with PQresultMemorySize() */
+#ifdef HAVE_PQRESULTMEMORYSIZE
+	/* Verbose error verbosity level (#set_error_verbosity).
+	 * The SQLSTATE mode includes only the error severity and the SQLSTATE error code, if one is available (if not, the output is like TERSE mode).
```

Comment:
Do these comments appear in the gem documentation?

L466 is identical to the comment for "VERBOSE mode" while L467 says "like TERSE mode."